app/views/search/_results.html.slim

```diff
@@ -0,0 +1,27 @@
+.col-xs-12.col-sm-9
+  .page-details__row--quarter
+    = form_tag search_path, method: :get do
+      .row
+        .col-xs-12.col-sm-6
+          h6
+            span= "Results for \"#{@query}\" "
+            span.gray-light
+              | &bull;
+              = " #{number_with_delimiter(@total_count, delimiter: ' ')}"
```

Comment:
I can change it but the current design reflects what I found in the Sketch files:

![screen shot 2560-04-27 at 12 14 44 pm](https://cloud.githubusercontent.com/assets/2647689/25468913/2af0a1c8-2b43-11e7-9685-6bdf74037f19.png)